psa-crypto-sys/Cargo.toml

```diff
@@ -18,6 +18,7 @@ links = "mbedcrypto"
 bindgen = "0.54.0"
 cc = "1.0.54"
 cmake = "0.1.44"
+walkdir = "2"
```

Comment:
We usually put the dependencies we use in the `README.md` file as well, for recognition and to check that they have a compatible license!

Reply:
Ahh gotcha, will fix. :)